src/o365/spo/commands/contenttype/contenttype-field-remove.ts

```diff
@@ -0,0 +1,307 @@
+import auth from '../../SpoAuth';
+// import { ContextInfo, ClientSvcResponse, ClientSvcResponseContents } from '../../spo';
```

Comment:
We shouldn't include commented out code. If we don't need it, let's remove it for clarity